app/modules/encounters/schemas.py

```diff
@@ -71,10 +71,9 @@ class Meta(BaseEncounterSchema.Meta):
 
 class AugmentedIndividualApiEncounterSchema(BaseEncounterSchema):
 
-    submitter = base_fields.Nested(
-        'BaseUserSchema', many=False, only=('full_name', 'guid')
-    )
-    owner = base_fields.Nested('BaseUserSchema', many=False, only=('full_name', 'guid'))
+    submitter = base_fields.Nested('PublicUserSchema', many=False)
```

Comment:
These were just wrong, the front end needs the same data in all cases to display the user info so just use the schema